src/unix/linux_like/mod.rs

```diff
@@ -1,3 +1,10 @@
+#[cfg(feature = "extra_traits")]
+use core::{
+    cmp::{Eq, PartialEq},
+    fmt::Debug,
+    iter::Iterator,
+};
```

Comment:
Seems this import style won't work on an old Rust.

Reply:
Should be fixed now.